### ISSUE TITLE
Admin Accessibility: Adding a media type

### ISSUE THREAD

**Issue by carlj11** [omeka/omeka-s#2183]
When you click an item such as "brittle end paper", the selection dialog closes, or is replaced by one which allows you to add a caption.

- the caption field should gain focus when this appears
- the caption field has no programmatically attached label (i.e. screen reader should announce "caption" when it gains focus)
